.github/workflows/getEnvironmentVariables.js

```diff
@@ -9,9 +9,12 @@ program
   .name("Approve PR")
   .description("Approve and merge PR if patch release")
   .option("--version <version>", "version of a project")
-  .option("--token <token>", "token fo login to cloud")
+  .option("--repo_token <token>", "token for login to cloud")
```

Comment:
You removed `--token` but you are still passing `--token` + the help text is saying it's a "cloud" token whereas it's a GitHub token.